granadilla/templates/granadilla/user_password.html

```diff
@@ -0,0 +1,25 @@
+{% extends "granadilla/base.html" %}
+{% load granadilla_tags %}
+{% load i18n %}
+
+{% block subtitle %} {{user.username}} {% endblock %}
+
+{% block breadcrumbs %}<a href="{% url "granadilla:index" %}">{% granadilla_title %}</a> &rsaquo; {{user.username}} &rsaquo; Change Password {% endblock %} 
+
+
+{% block content %}
+<form action="" method= "POST">
+<table class="module">
+{% csrf_token %}
+
+<caption>{{ user.username }}</caption>
+	    <tr>
+			{{form}}
+	    </tr>	    
+			<td>&nbsp;</td> <td><input type="submit" value="Change password"> </td>
```

Comment:
This HTML is not valid: there is a ``<td>`` outside a row?
Also, ``Change password`` should be marked for translation.